Adding in talks from 35th meetup and some historic meetups

AaronCritchley opened the pull request:
Will add more when I get time (hopefully you're OK with them being in seperate PR's).

Couldn't find a link to Calvin's slides, happy to update if they're provided.

calvingiles commented on meetups/index.md:
```diff
@@ -5,18 +5,29 @@ title: Meetups
-  * Saeed Amen on Using Python in financial markets
-  * Won Bae Suh on Arms for Sale: UK Bazzar in the 21st Century
+  * Richard J Brooker on [AI & ART - Style Transfer](https://www.youtube.com/watch?v=U1vb-NI9sKE)
+  * Saeed Amen on [Using Python in financial markets](https://www.meetup.com/PyData-London-Meetup/events/240813803/comments/481296769/)
```
I think a direct link to the slides would be good, maybe followed with a short link or set of links to the extra material. Happy for you to link to either the comment or a bunch of short links to the content directly.

My only concern with the comment is comments can get deleted. On the other hand, extra links might get added to it later. \shrug

kynan replied:
Agree with @calvingiles, let's not rely on anything on meetup being persistent :)

AaronCritchley replied:
Sure, will change tonight when I'm out the office. As far as I can see the slides haven't been uploaded (please correct me if I am wrong), I'll link to the github content and examples directly though!

kynan replied:
That's right, Saeed wants people to email him for the slides.

kynan commented:
@AaronCritchley just a general note: feel free to change things around completely! Maybe instead of a list we could have each meetup be an `h1` or `h2` and then have the talks even with abstract and speaker bio. Just an idea and only to say you have "free reign".

AaronCritchley commented:
@kynan thanks for the note, I'll definitely look into ways that we could change the page layout around :smile:

AaronCritchley commented:
Updated, addressed comments from @calvingiles and @kynan. I plan on writing out some concepts for the new page format sometime soon, but dependant on when I get some spare time.

Let me know if further changes are needed!

kynan commented:
Thanks!
